Docs/Configuration.tex

```diff
@@ -4543,10 +4543,12 @@ \subsection{Properties}\label{platforminfoprops}
   \item When disabled \texttt{Generic} section is unused.
   \end{itemize}
 
-  \textbf{Warning}: It is strongly discouraged set this option to \texttt{false}
-  when intending to update platform information. The only reason to do that is
-  when doing minor correction of the SMBIOS present and similar. In all other
-  cases not using \texttt{Automatic} may lead to hard to debug errors.
+  \textbf{Warning}: You are strongly discouraged from setting this option
```

Comment:
Please keep third-person pronouns here. i.e. `It is strongly discouraged to...`

Reply:
Apologies, I wouldn't have changed that part of the wording except that I was already helping in reviewing the current set of changes by @dakanji, where he seemed to be making the wording more like this. I'm off course happy to change that back to as it was. I'm not at my Mac now, but will do that along with other coding changes requested (once they have been, and assuming rest of PR is accepted in principle). Thanks.

Reply:
Okay will get to all of these asap - if I can do so, today.